shimmer/develop/what-is-shimmer/layer-1-landscape.md

```diff
@@ -0,0 +1,141 @@
+| Data structure                                                 | DAG(Tangle)                                                                                            | DAG(Tangle)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
+| Transaction execution                                          | Parallel                                                                                               | Parallel                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
+| Value transactions                                             | Yes                                                                                                    | Yes                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           |
+| [Store data](#store-data)                                      | Yes                                                                                                    | Yes, with Proof of Inclusion                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |
```

Comment:
Technically we also have PoI for IOTA 🤔

Reply:
practically as well?

Reply:
https://github.com/iotaledger/hornet/tree/production/plugins/poi


I would say so